### ISSUE TITLE
Incorrect manual download instructions for imagenet2012

### ISSUE THREAD

**Issue by mattdutson** [tensorflow/datasets#2158]
I am using TensorFlow 2.2 and TensorFlow Datasets 1.2.

The [imagenet2012 dataset documentation](https://www.tensorflow.org/datasets/catalog/imagenet2012) instructs the user to manually download `ILSVRC2012_img_train.tar` and `ILSVRC2012_img_val.tar` to `~/tensorflow_datasets/manual`. Even after doing this, a call to `tfds.load("imagenet2012")` attempts to download the dataset from the web, which results in an error due to the need for a login.

I have also tried moving the `.tar` files to `~/tensorflow_datasets/downloads/manual` and `~/tensorflow_datasets/downloads/manual/imagenet2012`, none of which have worked.

Not sure if this is a documentation issue or a bug.

**Comment by Conchylicultor** [comment]
TensorFlow Datasets 1.2 is very old. Could you try again with TFDS 3.1+ or tfds-nightly ?

**Comment by mattdutson** [comment]
@Conchylicultor Thanks for the reply. 1.2 is the most recent version of TFDS available through Anaconda. Didn't realize it was so out of date.

I was able to get this to work with TFDS 3.1, but only after moving the data to `~/tensorflow_datasets/downloads/manual`. This does not match the documentation, which says to place the data in `~/tensorflow_datasets/manual`.

**Comment by Conchylicultor** [comment]
> This does not match the documentation, which says to place the data in ~/tensorflow_datasets/manual.

Thank you for reporting. Sending a fix now.

**Comment by wilderfield** [comment]
How funny, I hit the same issue, and found the same fix as @mattdutson on the exact same day...
I still can't `python -m tensorflow_datasets.scripts.download_and_prepare --datasets=imagenet2012` to work though. I will open a separate issue.

**Comment by wilderfield** [comment]
> How funny, I hit the same issue, and found the same fix as @mattdutson on the exact same day...
> I still can't get `python -m tensorflow_datasets.scripts.download_and_prepare --datasets=imagenet2012` to work though. I will open a separate issue.

@mattdutson , @Conchylicultor, if you have a moment, does anything stand out to you in the error message I'm getting here: #2172 ?

**Comment by vijayphoenix** [comment]
Fixed by https://github.com/tensorflow/datasets/pull/2163.
Online Doc will be updated soon
